File: ClientApp/src/app/views/employee/model/emp-fingerprint.ts
export class EmpFingerprint {
    id : number = 0;
	empId : number | null = null;
	rightThumbFile : File | null = null;
	rightIndexFile : File | null = null;
	rightMiddleFile : File | null = null;
	rightRingFile : File | null = null;
	rightLittleFile : File | null = null;
	leftThumbFile : File | null = null;
	leftIndexFile : File | null = null;
	leftMiddleFile : File | null = null;
	leftRingFile : File | null = null;
	leftLittleFile : File | null = null;
	rightThumb : string = '';
	rightIndex : string = '';
	rightMiddle : string = '';
	rightRing : string = '';
	rightLittle : string = '';
	leftThumb : string = '';
	leftIndex : string = '';
	leftMiddle : string = '';
	leftRing : string = '';
	leftLittle : string = '';
    isActive : boolean = true;
}

File: Hrm.Domain/EmpFingerPrint.cs
using Hrm.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hrm.Domain
{
    public class EmpFingerPrint : BaseDomainEntity
    {
        public int Id { get; set; }
        public int? EmpId { get; set; }
        public string? RightThumb { get; set; }
	    public string? RightIndex { get; set; }
	    public string? RightMiddle { get; set; }
	    public string? RightRing { get; set; }
	    public string? RightLittle { get; set; }
	    public string? LeftThumb { get; set; }
	    public string? LeftIndex { get; set; }
	    public string? LeftMiddle { get; set; }
	    public string? LeftRing { get; set; }
	    public string? LeftLittle { get; set; }
        public bool IsActive { get; set; }

        public virtual EmpBasicInfo? EmpBasicInfo { get; set; }

    }
}
